northstar/src/runtime/state.rs

```diff
@@ -130,21 +134,44 @@ impl ProcessContext {
 impl<'a> State<'a> {
     /// Create a new empty State instance
     pub(super) async fn new(config: &'a Config, events_tx: EventTx) -> Result<State<'a>, Error> {
-        let mut repositories = Repositories::default();
+        let repositories = Repositories::default();
+        let mount_control = Arc::new(MountControl::new(&config).await.map_err(Error::Mount)?);
```

Comment:
Why do we need an Arc here?

Reply:
The `mount_control` is used in a `task::spawn` and needs to be cloned for that. Otherwise we would borrow `&self`.